backend/src/services/v2/release.ts

```diff
@@ -170,6 +170,28 @@ export async function updateRelease(user: UserDoc, modelId: string, semver: stri
   return updatedRelease
 }
 
+export async function newReleaseComment(user: UserDoc, modelId: string, semver: string, message: string) {
+  const release = await getReleaseBySemver(user, modelId, semver)
+
+  const comment = {
+    message,
+    user: user.dn,
+    createdAt: new Date().toISOString(),
+  }
+  const updatedRelease = await Release.findOneAndUpdate(
+    { _id: release._id },
+    {
+      $push: {
+        comments: comment,
+      },
+    },
+  )
+  if (!updatedRelease) {
```

Comment:
You might as well do this just after line 174 to save you waiting until you do another mongo query to find out of the release doesn't exist.